db_scripts/mosip_master/ddl/master-valid_document.sql

```diff
@@ -17,7 +17,7 @@
 CREATE TABLE master.valid_document(
 	doctyp_code character varying(36) NOT NULL,
 	doccat_code character varying(36) NOT NULL,
-	lang_code character varying(3) NOT NULL,
+	lang_code character varying(3),
```

Comment:
We should use alter table not change the create table.

Reply:
taken care